go/vt/vtgate/evalengine/fn_time.go

```diff
@@ -1249,6 +1254,53 @@ func (call *builtinLastDay) compile(c *compiler) (ctype, error) {
 	return ctype{Type: sqltypes.Date, Flag: arg.Flag | flagNullable}, nil
 }
 
+func fromDays(loc *time.Location, d int64) *datetime.Date {
+	// 3652424 days corresponds to maximum date i.e. 9999-12-31
+	// mysql returns 0000-00-00 for days below 366
+	if d > 3652424 || d < 366 {
```

Comment:
Can we put these values into named constants to describe them better?

Reply:
Removed the function completely using `MysqlDateFromDayNumber` instead.